assets/Vagrantfile

```diff
@@ -1,5 +1,10 @@
 $install_storageos = <<SCRIPT
 
+
+sudo apt-get -y purge docker-ce 
```

Comment:
How about avoiding this purge by not installing docker using vagrant provisioner in the first place? That would reduce the setup time.
Maybe remove https://github.com/storageos/storageos.github.io/pull/146/files#diff-b1c6364185dcedc82c38d557c660bf21R42

Reply:
I tested your point and it works well. I applied your suggestion as it is a way more elegant procedure. https://github.com/storageos/storageos.github.io/pull/146/commits/6ad72127c8d528f6d48d7eb0574efba255016905

Thank you!